frontend/src/views/WheresMyGene/components/GeneSearchBar/index.tsx

```diff
@@ -328,6 +328,8 @@ export default function GeneSearchBar({ onGenesChange }: Props): JSX.Element {
           }
           renderOption={(option) => option.name}
           onPaste={handlePaste}
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- TODO revisit lint errors
+          // @ts-ignore -- TODO revisit lint errors
```

Comment:
I was getting an error here when running `npm run lint`.

Reply:
Ah sorry for the trouble! I have a [fix](https://github.com/chanzuckerberg/single-cell-data-portal/pull/2018) for this after an SDS PR is merged later today!

Reply:
No worries at all! Thanks for the update, Timmy.